### PR TITLE
feat(ld-tabs): use new color vars

### PR DESCRIPTION
**Pull request by renet**
# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/liquid/components/ld-tabs/ld-tablist/ld-tablist.tsx [hunk 1]
```diff
@@ -123,6 +123,28 @@ export class LdTablist {
...
+      icon.classList.remove('ld-icon--sm', 'ld-icon--lg')
+      if (this.size === 'sm') {
+        icon.classList.remove('ld-icon--lg')
+        icon.classList.add('ld-icon--sm')
+      } else if (this.size === 'lg') {
+        icon.classList.remove('ld-icon--sm')
+        icon.classList.add('ld-icon--lg')
+      } else {
+        icon.classList.remove('ld-icon--sm', 'ld-icon--lg')
+      }
```
Comment (borisdiakur) [inline src/liquid/components/ld-tabs/ld-tablist/ld-tablist.tsx:144]: This is redundant.

```suggestion
      }
```

**Review by borisdiakur — CHANGES_REQUESTED** [review]

File: src/liquid/components/ld-tabs/ld-tablist/ld-tablist.tsx [hunk 2]
```diff
@@ -123,6 +123,28 @@ export class LdTablist {
...
+      if (this.size !== undefined) {
+        icon.setAttribute('size', this.size)
+      } else {
+        icon.removeAttribute('size')
+      }
+    })
+    this.el.querySelectorAll('.ld-icon').forEach((icon) => {
+      icon.classList.remove('ld-icon--sm', 'ld-icon--lg')
+      if (this.size === 'sm') {
+        icon.classList.remove('ld-icon--lg')
```
Comment (borisdiakur) [inline src/liquid/components/ld-tabs/ld-tablist/ld-tablist.tsx:137]:
```suggestion
```

File: src/liquid/components/ld-tabs/ld-tablist/ld-tablist.tsx [hunk 3]
```diff
@@ -123,6 +123,28 @@ export class LdTablist {
...
+        icon.classList.add('ld-icon--sm')
+      } else if (this.size === 'lg') {
+        icon.classList.remove('ld-icon--sm')
```
Comment (borisdiakur) [inline src/liquid/components/ld-tabs/ld-tablist/ld-tablist.tsx:140]:
```suggestion
```

**Review by borisdiakur — CHANGES_REQUESTED** [review]

File: src/liquid/components/ld-tabs/ld-tablist/ld-tablist.tsx (hunk 1 above)
Comment (renet) [reply src/liquid/components/ld-tabs/ld-tablist/ld-tablist.tsx:143]: I'll fix it the other way around, as this reduces unnecessary DOM manipulations.

**Review by borisdiakur — APPROVED** [review]

**Pull request merged**